org.eclipse.jdt.ui.tests/pom.xml

```diff
@@ -39,8 +53,9 @@
           <useUIThread>true</useUIThread>
           <includes>
             <include>org/eclipse/jdt/ui/tests/AutomatedSuite.class</include>
```

Comment:
I mean this one. It should also be the source name, right?

Reply:
That's why I said this seems to be undocumented. It is how jdt.ui is making use of tycho for years. It in fact makes tycho starting AutomatedSuite and nothing else.